arch/arm64/include/spinlock.h

```diff
@@ -83,6 +83,25 @@
  * -- Clear Exclusive access monitor (CLREX) This is used to
  *     clear the state of the Local Exclusive Monitor.
  */
+#if defined(CONFIG_TICKET_SPINLOCK)
```

Comment:
let's make the implementation work with other arch too

Reply:
I have refined the code and check ticket spinlock is unrelated architecture.
And test on https://github.com/apache/nuttx-apps/pull/2076
Please review.